licenses/github.com/otiai10/curr/LICENSE

```diff
@@ -0,0 +1,5 @@
+Copyright © 2015 Hiromu OCHIAI <otiai10@gmail.com>
+
+This work is free. You can redistribute it and/or modify it under the
+terms of the Do What The Fuck You Want To Public License, Version 2,
```

Comment:
I am pretty sure this license is explicitly banned within google (and likely others as well)

Reply:
> Can we drop the dependency?

This if from `kubernetes/test-infra`. I am not sure removing it is an (easy) option.